packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadListActionsRequired.tsx

```diff
@@ -63,28 +58,34 @@ export const DownloadListActionsRequired: React.FunctionComponent<
   const renderActionRequired = (actionRequiredCount: ActionRequiredCount) => {
     switch (actionRequiredCount.action.concreteType) {
       case 'org.sagebionetworks.repo.model.download.MeetAccessRequirement': {
-        const meetARAction: MeetAccessRequirement =
-          actionRequiredCount.action as MeetAccessRequirement
```

Comment:
No need for this case because I changed the type definitions so the type can be inferred when the concreteType matches that string.